lib/node_modules/@stdlib/stats/base/variancech/lib/ndarray.js

```diff
@@ -59,6 +64,11 @@ function variancech( N, correction, x, stride, offset ) {
 	var n;
 	var i;
 
+	// Check for accessor array:
+	if ( x.get ) {
```

Comment:
This isn't a sufficient check. Please follow what we do in other implementations supporting the accessor protocol.